.gitmodules

```diff
@@ -7,7 +7,7 @@
 	url = https://github.com/tommie/lx106-hal
 [submodule "esptool"]
 	path = esptool
-	url = https://github.com/pfalcon/esptool
+	url = https://github.com/themadinventor/esptool.git
```

Comment:
Ok, i see the reason. Does this mean that your esptool fork stays where it is not pulling changes from the origin? What about quite useful ones like cooking OTA images?